CONTRIBUTING.md

```diff
@@ -115,186 +116,6 @@ Minor refactoring, including incorporating changes to Keystone APIs, would not w
 
 Generally, these guidelines are in place so that we don't spam consumers with version upgrades that don't provide value. They are subjective however, and not "one size fits all" so if you're not sure whether a change warrants a version bump, ask for advice in the PR.
 
-### Release Guidelines
```

Comment:
I intend to review the rest of this document soon, but for now, this is easily removed